h2o-py/tests/testdir_algos/automl/pyunit_automl_event_log.py

```diff
@@ -24,8 +24,7 @@ def test_event_log():
     assert int(aml.training_info['stop_epoch']) > int(aml.training_info['start_epoch'])
     stop_dt = dt.datetime.fromtimestamp(int(aml.training_info['stop_epoch']))
     now = dt.datetime.now()
-    assert stop_dt.day == now.day
-    assert stop_dt.hour == now.hour
+    assert abs(stop_dt - now) < dt.timedelta(minutes=15)
```

Comment:
even if `15 min` looks like an excessive delta (the test itself is supposed to last a few seconds…), I think it's kind of ok, the only goal of the previous assertions was to verify that the`epoch` was in the correct format, here Unix timestamp instead of millis.

The duration assertion was really done below.

Reply:
> wrong handling of time zones

unix epoch is always UTC.
Both `dt.datetime.fromtimestamp` and `dt.datetime.now` will return an object on the local timezone, so there shouldn't be any timezone issue.

Reply:
I wasn't really sure why there was the comparison of hours and days. I know we have `h2o.setTimezone(...)` so it was the only reason that I could think of for comparing hours (different time zone on client vs h2o) but looking at `ai.h2o.automl.events.EventLogEntry` it seems that we really use UTC no matter what the user sets. 

@sebhrusen Is this change PR ok or should I change anything?

Reply:
can you please just add a comment explaining this assertion as at first sight the 15min delta looks like it's not testing much: honestly if it takes 15min between the end of training on backend and the completion on client side, then there is a very serious problem somewhere.
I'd set the delta to `1min`, and mention that it's only verifying that `stop_epoch` was a unix epoch.

Reply:
Sure. Thanks!